Feature/themes and rqdata (#11)
hCaptcha/react-native-hcaptcha: MAINTAINER.md

```

Solution: updade `com.android.tools.build:gradle` version in `./android/build.gradle`

---

Problem:

Infinite error logs like:

```
WARN     Sending `didSendNetworkData` with no listeners registered.
WARN     Sending `didReceiveNetworkResponse` with no listeners registered.
WARN     Sending `didReceiveNetworkData` with no listeners registered.
WARN     Sending `didCompleteNetworkResponse` with no listeners registered.
ERROR    Invariant Violation: No callback found with cbID xxxxx and callID yyyyy for module <unknown>. Args: '[zzzz]'

```

Solution: delete `node_modules` in `react-native-hcaptcha`.

> Issue related to missmatch `react-native` versions in a test app and `react-native-hcaptcha`

Reviewer: ```suggestion
This issue is related to mismatched `react-native` versions in the test app vs. `react-native-hcaptcha`
```
